tsconfig.json

```diff
@@ -5,24 +5,19 @@
     "target": "esnext",
     "lib": ["dom", "dom.iterable", "esnext"],
     "jsx": "react-jsx",
-    "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "isolatedModules": true,
     "resolveJsonModule": true,
     "noEmit": true,
     "incremental": true,
     "sourceMap": true,
-    "declaration": true,
-    "declarationMap": true,
```

Comment:
Moved to `tsconfig` files in `packages` subfolders, where it is actually needed. This was causing issues in the `storybook` project.